tsconfig.json

```diff
@@ -26,16 +26,11 @@
     "noEmit": true,
 
     "paths": {
-      "@glimmer/test-helpers": ["../test-helpers/index.ts"]
+      "*": ["*"]
     }
   },
   "include": [
-    "packages/*/*.d.ts",
-    "packages/*/index.ts",
-    "packages/*/lib/**.ts",
-    "packages/@glimmer/*/*.d.ts",
-    "packages/@glimmer/*/index.ts",
-    "packages/@glimmer/*/lib/**.ts"
+    "packages/**/*.ts"
```

Comment:
@krisselden That's correct, those packages are "source-only" and all builds happen in the root `dist/`.